lms/djangoapps/course_home_api/progress/v1/serializers.py

```diff
@@ -5,46 +5,62 @@
 from rest_framework.reverse import reverse
 
 
-class GradedTotalSerializer(serializers.Serializer):
-    earned = serializers.FloatField()
-    possible = serializers.FloatField()
+class CourseGradeSerializer(serializers.Serializer):
+    percent = serializers.FloatField()
+    is_passing = serializers.SerializerMethodField()
```

Comment:
Ooh thank you for this. I wasn't sure if there was a better way to do this.

Also, I wanted to change this from 'passed' to 'is_passing' because the passing status can vary (ie. you might be passing at one point, but if an assignment is added you may no longer be passing), and the past tense of 'passed' felt too set in stone.

Reply:
Yeah, I see what you're saying. Not sure how often that happens though, but /shrug. Not my hill to die on